.github/workflows/ci.yml

```
@@ -0,0 +1,92 @@
+
+      - name: Build Solution
+        run: dotnet build ${{ env.SOLUTION_PATH }} -c Release --no-restore
+
+      - name: Build Unit Tests
+        run: dotnet build src/MeetlyOmni.Unit.tests/MeetlyOmni.Unit.tests.csproj -c Release
+
+      - name: Unit Test
+        run: dotnet test src/MeetlyOmni.Unit.tests/MeetlyOmni.Unit.tests.csproj -c Release --no-build --no-restore
+
+      - name: Wait Testdb ready
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2034:warning:1:1: i appears unused. Verify use (or export if used externally)

(shellcheck)